src/targets/crates.ts

```diff
@@ -202,9 +209,26 @@ export class CratesTarget extends BaseTarget {
       '--manifest-path',
       crate.manifest_path,
     ];
-    return spawnProcess(CARGO_BIN, args, {
-      env: { ...process.env, CARGO_REGISTRY_TOKEN: this.cratesConfig.apiToken },
-    });
+    const env = {
+      ...process.env,
+      CARGO_REGISTRY_TOKEN: this.cratesConfig.apiToken,
+    };
+
+    logger.info(`Publishing ${crate.name}`);
+    for (let i = 0; i <= 1; i++) {
```

Comment:
I think this for loop construct makes it harder to read the code than repeating the `return await spawnProcess(CARGO_BIN, args, { env });` part. If you don't want to repeat that, we can abstract that out  into its own function.

Or at least we should define `MAX_RETRIES=1` and do `i <= MAX_RETRIES`.

Reply:
Good point, I’ll repeat for now.